kvmagent/kvmagent/test/utils/vm_utils.py

```
@@ -642,6 +727,12 @@ def del_vm_scsi_controller(vm_uuid, iothread):
     return VM_PLUGIN.del_scsi_controller(misc.make_a_request(cmd.to_dict()))
 
 
+@misc.return_jsonobject()
+def del_vm_scsi_controller(vm_uuid):
```

Ruff (0.12.2)

Redefinition of unused `del_vm_scsi_controller` from line 720

(F811)